vmdb/app/models/service_template_orchestration.rb

```diff
@@ -0,0 +1,54 @@
+class ServiceTemplateOrchestration < ServiceTemplate
+  def create_subtasks(_parent_service_task, _parent_service)
+    # no sub task is needed for this service
+    []
+  end
+
+  def deploy_orchestration_stack(options)
+    params = {}
+    options.each do |attr, val|
+      if attr.start_with? 'dialog_param_'
+        params[attr['dialog_param_'.size..-1]] = val
+      elsif attr.start_with? 'password::dialog_param_'
+        params[attr['password::dialog_param_'.size..-1]] = MiqPassword.decrypt(val)
```

Comment:
These parameters look very AWS specific.  Should this class be further derived into `ServiceTemplateOrchestrationAmazon < ServiceTemplateOrchestration < ServiceTemplate`